tools/Azure.Mcp.Tools.Monitor/src/Commands/WebTests/WebTestsUpdateCommand.cs

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine.Parsing;
+using Azure.Mcp.Core.Commands;
+using Azure.Mcp.Core.Models.Option;
+using Azure.Mcp.Tools.Monitor.Models.WebTests;
+using Azure.Mcp.Tools.Monitor.Options;
+using Azure.Mcp.Tools.Monitor.Options.WebTests;
+using Azure.Mcp.Tools.Monitor.Services;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Mcp.Tools.Monitor.Commands.WebTests;
+
+public sealed class WebTestsUpdateCommand(ILogger<WebTestsUpdateCommand> logger) : BaseMonitorWebTestsCommand<WebTestsUpdateOptions>
+{
+    private const string CommandTitle = "Update a web test in Azure Monitor";
+
+    private readonly ILogger<WebTestsUpdateCommand> _logger = logger;
+
+    public override string Name => "update";
+
+    public override string Description =>
+        """
+        Updates an existing standard web test in Azure Monitor. Ping/Multstep web tests are deprecated and are not supported.
+        Returns the updated web test details.
+        """;
+
+    public override string Title => CommandTitle;
+
+    public override ToolMetadata Metadata => new()
+    {
+        Destructive = true,
+        Idempotent = true,
+        OpenWorld = false,
+        ReadOnly = false,
+        LocalRequired = false,
+        Secret = false
+    };
+
+    protected override void RegisterOptions(Command command)
+    {
+        base.RegisterOptions(command);
+
+        // Add required options
+        command.Options.Add(MonitorOptionDefinitions.WebTest.WebTestResourceName);
+        command.Options.Add(OptionDefinitions.Common.ResourceGroup.AsRequired());
+
+        // Add optional options - for update, most options become optional
+        command.Options.Add(MonitorOptionDefinitions.WebTest.AppInsightsComponentId.AsOptional());
```

Comment:
You shouldn't need AsOptional or AsRequired unless you are changing the default requiredness